src/main/webapp/app/iris/base-chatbot/chat-status-bar/chat-status-bar.component.ts

```diff
@@ -16,7 +21,7 @@ export class ChatStatusBarComponent implements OnChanges {
     displayedSubText?: string;
     style?: string;
 
-    @Input() stages?: IrisStageDTO[] = [];
+    stages = input<IrisStageDTO[]>();
```

Comment:
If the input is definitly needed for the component then I would suggest to make it required. This way you don't need to do null checks afterwards.
```suggestion
    stages = input.required<IrisStageDTO[]>();
```

Reply:
In `iris-base-chatbot.component.ts` the stages are defined as `stages?IrisStageDTO[] = [];`, so as I understand, we have cases where the stages are undefined.